framework/mongo/utils.py

```diff
@@ -60,16 +61,78 @@ def wrapper(cls):
         return cls
     return wrapper
 
-
-def get_or_http_error(Model, pk, allow_deleted=False):
-    instance = Model.load(pk)
+def get_or_http_error(Model, pk_or_query, allow_deleted=False, display_name=None):
```

Comment:
In general, I agree. In this case an Upper case param name is helpful to indicate that the function takes a class rather than in instance.